ProteoformSuiteInternal/ExcelReader.cs

```diff
@@ -0,0 +1,126 @@
+﻿using DocumentFormat.OpenXml.Packaging;
```

Comment:
I removed this class a while back. Could you check to see if it's still being used in your code and why? Looks like most of this can be moved to ComponentReader to match the master branch.